### DISCUSSION TITLE
利用google cloud 的机器无法利用终端远程连接

### DISCUSSION THREAD

**Discussion by Marcus0910** [1Panel-dev/1Panel#7162] [Q&A]
### 联系方式

wechat: kahungg

### 1Panel 版本

v1.10.21-lts

### 问题描述

利用google cloud 的机器无法利用终端远程连接 port 22 什么的都打开了  用ssh 软件也进不去 ， 有人知道原因吗

### 重现步骤

可以用QQ远程看我电脑

### 期待的正确结果

能进去终端

**Comment by wanghe-fit2cloud** [comment]
使用 `IP:Port` 方式是否可以正常访问？
> 如果是反代方式访问，可以参考文档：https://1panel.cn/docs/faq/system_function/

**Reply by Marcus0910** [reply]
一样不行 ， 连上去登录root 会直接报错。 找咸鱼找了好多人看都弄不好 害

**Reply by wanghe-fit2cloud** [reply]
麻烦贴一下错误信息。
> 如果是超时问题，建议重点检查是否由网络原因导致。

**Reply by Marcus0910** [reply]
![Screenshot 2024-11-24 144352](https://github.com/user-attachments/assets/b2298021-b343-4fc3-9b01-5d86ff5f7412)

**Reply by Marcus0910** [reply]
一直就是弹出来这个链接不可用， 但是google cloud 本来就是利用ssh 技术链接

**Reply by wanghe-fit2cloud** [reply]
在 `主机` 页面添加该服务器的 IPv4 地址（请勿使用 `127.0.0.1`），测试是否能成功添加并正常访问终端。
